app/views/errors/not_found.html.erb

```diff
@@ -0,0 +1,12 @@
+
+
+<br><h1 class="card-title text-center "></h1><br>
+
+    <div class="col-sm-9 col-md-4 col-lg-7  mx-auto">
+        <div class="card border-secondary mb-3" >
+            <div class="card-header text-center"><h5>ERROR</h5></div>
+            <div class="card-body text-center" style= "background-color:white;">
+              <p>The page you are looking for does not exist. You may have mistyped the address or the page may have moved. Click <a href="/" class="card-link">here<a> to go back to the Obituary Hub Homepage. </p>
+        </div>
```

Comment:
Nice! hopefully we never get to actually see this!

Reply:
looking good Sage!